libs/parser-sql/src/postgresParser.ts

```
@@ -0,0 +1,1652 @@
+function tokenPosition(token: IToken): TokenPosition {
+    return {
+        offset: {start: pos(token.startOffset), end: pos(token.endOffset)},
+        position: {
+            start: {line: pos(token.startLine), column: pos(token.startColumn)},
+            end: {line: pos(token.endLine), column: pos(token.endColumn)}
+        }
+    }
+}
+
+function pos(value: number | undefined): number {
+    return value !== undefined && !isNaN(value) ? value : defaultPos
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)